gapis/api/vulkan/resources.go

```diff
@@ -1147,6 +1147,36 @@ func commonShaderDataGroups(ctx context.Context,
 							currentSetData = append(currentSetData, api.CreatePoDDataValue("", "-"))
 							currentSetData = append(currentSetData, api.CreatePoDDataValue("", "-"))
 
+						case VkDescriptorType_VK_DESCRIPTOR_TYPE_INPUT_ATTACHMENT:
+							descInfo := bindingInfo.ImageBinding().Get(i)
+
+							currentSetData = append(currentSetData, api.CreatePoDDataValue("", "-"))
+
+							viewHandle := descInfo.ImageView()
+							viewPath := path.NewField("ImageViews", resolve.APIStateAfter(path.FindCommand(cmd), ID)).MapIndex(viewHandle).Path()
+
+							imageView, _ := vkState.ImageViews().Lookup(viewHandle)
+							imageViewPath := cmd.ResourceAfter(path.NewID(resources[imageView.Image().ResourceHandle()])).Path()
+
+							paths := []*path.Any{viewPath, imageViewPath}
+
+							if !fb.IsNil() {
+								for _, v := range fb.ImageAttachments().Keys() {
+									fbViewHandle := fb.ImageAttachments().Get(v).VulkanHandle()
+
+									if fbViewHandle == viewHandle {
+										paths = append(paths, cmd.FramebufferAttachmentAfter(v).Path())
+										break
+									}
+								}
+							}
+
+							currentSetData = append(currentSetData, api.CreateLinkedDataValue("url", paths, api.CreatePoDDataValue("VkImageView", viewHandle)))
+
```

Comment:
Why all those extra empty lines?

Reply:
I guess I can get rid of the empty line at 1175. 1162 and 1173 separates the logic to find the input attachment index, I guess.